lib/onc_certification_g10_test_kit/smart_standalone_patient_app_group.rb

```diff
@@ -131,14 +131,17 @@ class SmartStandalonePatientAppGroup < Inferno::TestGroup
           inputs: {
             requested_scopes: { name: :standalone_requested_scopes },
             received_scopes: { name: :standalone_received_scopes }
+          },
+          options: {
+            scope_version: :v1
```

Comment:
Applying the preset overwrites the scopes with v1 style scopes

Reply:
I don't think that is going to change before release.